### PR TITLE
Added favicon image and included in base.html

### PR DESCRIPTION
**Pull request by imskr**

### CODE REVIEW

**Comment by mohitkh7** [comment]
This is not rendering favicon image as expected.

**Comment by mohitkh7** [comment]
Make a new folder `images` inside `static` folder present in root directory.
Store this favicon.png file inside that newly created `static\images` folder.

File: learn/templates/base.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	{% endblock %}
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	
+	<link rel="shortcut icon" type="image/png" href="favicon.png">
```
Comment (mohitkh7) [inline learn/templates/base.html]: change href address to `"{% static 'images/favicon.png %}"`
ref - https://docs.djangoproject.com/en/2.1/howto/static-files/

Comment (imskr) [reply learn/templates/base.html]: Ok I will edit it

**Comment by imskr** [comment]
ok i'm closing it and i will create fresh PR

**Pull request closed without merging**